pom.xml

```diff
@@ -195,6 +201,12 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
                 <version>3.1.2</version>
+                <configuration>
```

Comment:
Can we add a short comment describing why we need this change?

Reply:
Done.